### PR TITLE
Add com.github.amikha1lov.RecApp

### PR DESCRIPTION
**Pull request by amikha1lov**
<p align="center">
  
</p>

### CODE REVIEW

**Comment by amikha1lov** [comment]
bot, build com.github.amikha1lov.RecApp

File: com.github.amikha1lov.RecApp.yaml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  - /share/pkgconfig
+modules:
+  - shared-modules/glu/glu-9.json
+
+  - name: recapp
+    builddir: true
+    buildsystem: meson
+    sources:
+      - type: dir
+        path: .
```
Comment (tim77) [inline com.github.amikha1lov.RecApp.yaml:35]:
```suggestion
      - type: git
        tag: v1.0.1
        commit: ad0bf3ead9241233afbd142f7cdf5cf8b3511aaa
        url: https://github.com/amikha1lov/RecApp/
```

**Comment by amikha1lov** [comment]
bot, build com.github.amikha1lov.RecApp

**Comment by amikha1lov** [comment]
bot, build com.github.amikha1lov.RecApp

**Comment by amikha1lov** [comment]
bot, build com.github.amikha1lov.RecApp

**Comment by amikha1lov** [comment]
bot, build com.github.amikha1lov.RecApp

**Comment by amikha1lov** [comment]
@nedrichards for some reason, the repository was not created.

**Comment by amikha1lov** [comment]
Hello. I lost write access to 2 repositories recapp and avocode, how can I restore it?

**Comment by amikha1lov** [comment]
@nedrichards @bilelmoussaoui please help

**Comment by barthalion** [comment]
@amikha1lov I've sent you invitations to both repos.

**Comment by amikha1lov** [comment]
Thanks a lot

**Pull request closed without merging**